### PR TITLE
(FM-3747) Prepare Supported release 1.2.0

### PR DESCRIPTION
**Pull request by ferventcoder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-##2015-07-28 - Supported Release 1.1.0 
+##2015-10-14 - Supported Release 1.2.0
+###Summary
+
+Detect more pending reboot scenarios.
+
+###Features
+* Pending reboot - detect computer rename (MODULES-2657)
```
Comment (Iristyle) [inline CHANGELOG.md]: Do we normally link to these?

Comment (ferventcoder) [inline CHANGELOG.md]: Not always, but I will :)

File: metadata.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
         "Server 2012",
         "Server 2012 R2",
         "7",
-        "8"
+        "8",
+        "10"
```
Comment (Iristyle) [inline metadata.json]: Just discussed with @ajjohnsonpdx - we think we should hold off on this for the moment given we don't have cells in the matrix to test.

Comment (ferventcoder) [inline metadata.json]: That's fair. Will remove

Comment (ferventcoder) [inline metadata.json]: yanked.

File: CHANGELOG.md (hunk 1 above)
Comment (ferventcoder) [inline CHANGELOG.md]: Linked

**Pull request merged**